front-end/src/app/shared/components/transaction-type-base/transaction-contact.utils.ts

```diff
@@ -73,6 +73,9 @@ export class TransactionContactUtils {
   ): any[] {
     return Object.entries(contactConfig)
       .map(([field, property]: string[]) => {
+        // For Senate, 00 is used for district but not displayed in form, treated as NULL. It cannot be changed.
```

Comment:
Good find!  Looks like this is coming from the openfec data.  Let's fix this closer to the source.  What about in onFecApiCandidateLookupDataSelect().  We're doing something similar where we set the district to '' when the state is US.  Same idea but we could also include if the office is `S` to the condition

This would also fix the case where someone makes the contact using a lookup on the contact page, it won't result in us saving 00 in the district